data-intake/planning-data-load/planSchema.py

```diff
@@ -0,0 +1,114 @@
+from dataclasses import dataclass, field
+from typing import List, Optional
+
+
+# Data classes from Planning Data Load API
+
+@dataclass
+class PlanItem:
```

Comment:
Visier Python-SDK is generated from openapi-specifications. One of the advantages is that all DTOs are generated automatically.
https://github.com/visier/python-sdk/blob/main/src/visier_api_data_in/visier_api_data_in/models :

plan_data_load_change_dto.py
plan_data_load_change_list_dto.py
plan_data_load_error_dto.py
plan_data_upload_response_dto.py
plan_row_data_load_response_dto.py